edgelet/edgelet-docker/src/notary.rs

```diff
@@ -94,6 +97,39 @@ pub fn notary_init(homedir: &Path, hostname: &str, path: &PathBuf) -> Result<Pat
     Ok(config_file_path)
 }
 
+pub fn notary_lookup(
+    notary_auth: &str,
+    image_gun: &str,
+    tag: &str,
+    config_path: &str,
+    lock: tokio::sync::lock::LockGuard<()>,
+) -> impl Future<Item = String, Error = Error> {
+    Command::new("notary")
+        .env("NOTARY_AUTH", notary_auth)
+        .arg("lookup")
+        .args(&[image_gun, tag])
+        .args(&["-c", config_path])
+        .output_async()
+        .then(|notary_output| {
+            drop(lock);
+            let notary_output = notary_output.with_context(|_| {
+                ErrorKind::LaunchNotary("could not spawn notary process".to_owned())
```

Comment:
Can we open a tracking item to improve iotcheck for this kind of issue: docker content trust is enabled; however notary client is not setup/running correctly?

Reply:
Good point. Will do